cmd/whoami.go

```diff
@@ -46,7 +45,8 @@ func newWhoAmICmd() *cobra.Command {
 				return err
 			}
 
-			fmt.Println(name)
+			fmt.Println(fmt.Sprintf("User: %s", name))
+			fmt.Println(fmt.Sprintf("Backend URL: %s", b.URL()))
```

Comment:
I think it's fine - but I will note that I believe this is there first time we are using the word "Backend" in any user-facing context.  For example, we do not currently use that word in the `pulumi login` help text (and surprisingly we actually avoid using any word for the concept, just referring to it as a URL).

I think it's reasonable though to start using the word "backend" explicitly, and we may even want to update those login docs to use this term as well.

I would love @ellismg's thoughts on this as well though.